cmd/bosun/web/static/js/0-bosun.ts

```diff
@@ -78,6 +78,21 @@ bosunApp.config(['$routeProvider', '$locationProvider', function($routeProvider:
 		otherwise({
 			redirectTo: '/',
 		});
+	$httpProvider.interceptors.push(function($q) {
```

Comment:
The goal here is do enable miniprofiler only if there's a human in front of it. So anything that uses the API (like scollector and other /api/put or /api/metadata/put agents) will not trigger miniprofiler. This was causing out of memory issues with the raintank folks.